target/linux/ar71xx/base-files/lib/ar71xx.sh

```diff
@@ -287,11 +287,17 @@ tplink_board_detect() {
 	"360000"*)
 		model="TP-Link TL-WDR3600"
 		;;
-	"3C0001"*)
+	"3c0001"*)
 		model="OOLITE"
+
+		[ "$hwid" = "3c00010c" -a "$mid" = "00000001" ] && {
+			model="MINIBOX_V3.2"
+			hwver=""
+		}
 		;;
-	"3C0002"*)
+	"3c0002"*)
 		model="MINIBOX_V1"
+		hwver=""
```

Comment:
and this to v1 ?

Reply:
Yes, hwver on V1 was missing so I added it for coherence (someone on #lede-dev told me to do so)